tests/test-matpoly-mult.C

```diff
@@ -175,7 +175,7 @@ bool debug_midpgen_dlp(const Field& fld,  RandIter& Gen) {
 template<typename Field>
 bool launchTest(const Field& F, size_t n, long b, long d, long seed){
 	bool ok=true;
-	typename Field::RandIter G(F,b,seed);
+	typename Field::RandIter G(F,seed,b);
```

Comment:
b is a bitsize: should use a samplesize(1); samplesize<<b;

Reply:
done